src/loac/loac_compute.h

```diff
@@ -0,0 +1,517 @@
+#pragma once
+
+#include <vector>
+#include <algorithm>
+#include <limits>
+#include <iostream>
+#include <iomanip>
+#include <unordered_map>
+#include <random>
+#include <queue>
+
+#include "../kmers.h"
+#include "../unionfind.h"
+
+#define size_k_max uint8_t
```

Comment:
It is a better practice to define custom types in c++ using typedefs
```suggestion
typedef uint8_t size_k_max;
```